scipy/optimize/_minimize.py

```diff
@@ -112,6 +114,12 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         to select a finite difference scheme for numerical estimation of the
         gradient with a relative step size. These finite difference schemes
         obey any specified `bounds`.
+
+        .. note::
+           the `grad` keyword was formerly referred to as `jac`. As of SciPy
+           1.11.0, either may be used, but consider using `grad` to avoid
+           confusion with the constraint Jacobian.
```

Comment:
I think it is slightly misleading to say either may be used given that `jac` is deprecated and will be removed in the future

Reply:
That was written before I planned to deprecate it. I mentioned in the top post that I plan to change this to a deprecated admonition.